added steps to use analyzed span estimator (#6013)
DataDog/documentation: content/en/account_management/billing/usage_control_apm.md

Prices reflect annual billing. Contact [Sales][5] or your [Customer Success][6] Manager to discuss volume discounts for your account.

### Analyzed Span Estimator
[Analyzed Span Estimator][7] is designed to help you decide which services to configure with App Analytics while keeping usage and cost in your control.

> Steps to estimate the total number of Analyzed Spans expected to be sent per service per day or per month:

Reviewer: ```suggestion
To estimate the total number of Analyzed Spans expected to be sent per service per day or per month, follow these steps:
```

> 1. Enable APM for all the Hosts that you want to estimate the Volume of Analyzed Spans for.

Reviewer: ```suggestion
1. Enable APM on all the hosts for which you want to estimate the volume of Analyzed Spans.
```

> 2. On the [Estimator screen][7], select the services that you want to turn APM Analytics on.

Reviewer: ```suggestion
2. In the [*Analyzed Span Estimator* view][7], select the services that you want to turn APM Analytics on.
```

> 3. Total Estimated Volume denotes the Estimated Total Volume of Analyzed Spans for all services per day and per month. Each row corresponding to the service represents the Estimated Volume of Analyzed Spans for that service per day.

Reviewer: ```suggestion
3. Total Estimated APM Volume denotes the estimated total volume of analyzed spans for all services, per day and per month. Each row corresponding to the service represents the estimated volume of analyzed spans for that service, per day.
```

{{< img src="tracing/faq/apm_span_estimator.png" alt="Analyzed Span Estimator">}}

> 4. Total cost can be estimated by multiiplying the Total Volume * [Analyzed Span Retention Pricing][8]

Reviewer: ```suggestion
4. To estimate total cost, multiply the total volume * [analyzed span retention pricing][8]
```

> Ex. 1,750,000,000 Analyzed Spans per month for 15 days (default retention)

Reviewer: ```suggestion
For example, if you have 1,750,000,000 analyzed spans per month for 15 days (default retention), you can calculate:
```
